roadmap/implementers-guide/src/runtime/hrmp.md

```diff
@@ -1,6 +1,6 @@
-# Router Module
+# HRMP Module
 
-The Router module is responsible for all messaging mechanisms supported between paras and the relay chain, specifically: UMP, DMP, HRMP and later XCMP.
+A module responsible for HRMP. See [Messaging Overview](../messaging.md) for more details.
```

Comment:
Do links work like this? Asking for a friend..

Reply:
What's wrong with it? Genuine question

Reply:
Wasn't sure if the link targets have to end in `.html`, but they don't, that's only relevant for rust struct targets i.e. `[foo is great](struct.Foo.html)`

Reply:
Actually, mdbook supports both notations! But I prefer and encourage `*.md`, because it also works when you are viewing the docs on github